index.html

```diff
@@ -125,7 +125,7 @@ <h3>Profesional Art Printing Data</h3>
                     </ul>
                 </div>
                 <div class="card-body-bottom">
-                <button class="project-button" type="button"> See Project</button>
+                <button class="project-button" type="button" data-modal-target="#modal"> See Project</button>
               </div>
             </div>
               </div>
```

Comment:
Popup for only the first card is working. Kindly fix the issue for the rest of cards. Also the `see project` button is missing.
![image](https://user-images.githubusercontent.com/33268927/174053559-44e24c5e-d7f6-4fd5-b145-28b10e354511.png)

Reply:
Hi Muneeb, see project button was not included in the rest of the cards in Figma Template. Do you still want me to add it?